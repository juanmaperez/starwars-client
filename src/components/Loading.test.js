import React from 'react'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import toJson from 'enzyme-to-json'
import Loading from './Loading'


configure({ adapter: new Adapter() })

describe('<Loading />', () => {
  it('Loading snapshot matches', () => {
    const wrapper = shallow(<Loading />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})