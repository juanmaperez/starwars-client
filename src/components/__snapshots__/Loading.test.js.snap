// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> Loading snapshot matches 1`] = `
<styled.div>
  <div
    className="vader"
  >
    <div
      className="shadow"
    />
    <div
      className="head"
    >
      <div
        className="helmet"
      >
        <span
          className="left"
        />
        <span
          className="right"
        />
      </div>
      <div
        className="eyes"
      >
        <span
          className="left"
        />
        <span
          className="right"
        />
      </div>
      <span
        className="grill"
      >
        <span
          className="left"
        />
        <span
          className="center"
        />
        <span
          className="right"
        />
      </span>
      <span
        className="mask"
      >
        <span
          className="top"
        />
        <span
          className="left"
        />
        <span
          className="center"
        />
        <span
          className="right"
        />
      </span>
      <span
        className="line"
      />
    </div>
    <div
      className="torso"
    >
      <span
        className="neck"
      >
        <span
          className="left"
        />
        <span
          className="center"
        />
        <span
          className="right"
        />
        <span
          className="bottom"
        />
      </span>
      <span
        className="belt"
      >
        <span
          className="center"
        />
      </span>
      <div
        className="plate"
      >
        <span
          className="red_top"
        />
        <span
          className="red_center"
        />
        <span
          className="red_bottom"
        />
        <span
          className="blue"
        />
        <span
          className="gray"
        />
      </div>
    </div>
    <div
      className="hand left"
    >
      <span
        className="hand"
      />
    </div>
    <div
      className="hand right animation-right"
    >
      <span
        className="hand"
      />
    </div>
    <div
      className="legs"
    >
      <span
        className="left"
      />
      <span
        className="right"
      />
    </div>
    <div
      className="boots"
    >
      <span
        className="left"
      />
      <span
        className="right"
      />
    </div>
    <div
      className="sword animation-left"
    >
      <span
        className="handle"
      />
      <span
        className="light"
      />
    </div>
  </div>
  <div
    className="loading"
  >
    Loading...
  </div>
</styled.div>
`;
